src/config/settings.py

```diff
@@ -21,3 +21,68 @@
 AVAILABLE_IMPORTS = ["sonarqube"]
 
 AVAILABLE_GEN_FORMATS = ["csv", "json"]
+
+SUPPORTED_MEASURES = [
```

Comment:
Why is SUPPORTED_MEASURES inplemented in cli? In that sense the service would also have to have a SUPPORTED_MEASURES constant?

Reply:
Bem, era anteriormente realizado o import deste constante via core, pois o parser estava armazenado neste repositório.
Com a criação do novo parser, tendo em vista generalizar as fontes de entradas e valores retornados, nos decidimos que o parser iria retornar 100% das métricas que fossem passadas pelo arquivo, se estas métricas diferirem de None, é a aplicação iria utilizar as métricas que ela soubesse lidar(tendo em vista que para implementação de uma nova medida o valor já estaria sendo passado para a aplicação, sendo necessário implementar somente o cálculo).
Sabendo disso, e tendo em vista que o parser não sabe qual medida a cli, ou a aplicação externa sabe calcular, foi decidido passar responsabilidade de quais medidas o CLI sabe lidar para a própria CLI.

Reply:
ok